Sources/PayPalNativeCheckout/PayPalUIInterfaces/CheckoutProtocol.swift

```diff
@@ -12,7 +12,7 @@ protocol CheckoutProtocol {
     typealias CancelCallback = () -> Void
     typealias ErrorCallback = (PayPalCheckoutErrorInfo) -> Void
 
-    static func set(config: CoreConfig, returnURL: String)
+    static func set(config: CoreConfig)
```

Comment:
I dont see that the `returnUrl` is passed to `Checkout`, is it not needed on iOS?

Reply:
Return URL is no longer needed on iOS - I don't think it's needed on Android either (at least was not for the Braintree integration)

Reply:
it's working without return url